app/helpers/application_helper.rb

```diff
@@ -13,4 +13,17 @@ def bootstrap_alert_class_for(flash_type)
       flash_type.to_s
     end
   end
+
+  def error_messages_for(model)
+    return unless model.errors.any?
+
+    content_tag(:div, class: %w(alert alert-danger), role: %w(alert)) do
+      concat(content_tag(:h2, "There's a problem", class: %w(alert__heading)))
+      concat(content_tag(:ul) do
+        model.errors.full_messages.each do |message|
+          concat(content_tag(:li, message))
+        end
+      end)
+    end
```

Comment:
I like the succinctness of invoking it with `error_messages_for(..)` instead of `render partial: 'errors', model: the_model`. The nested `content_tags` are nasty though.

Any opinion @andrewgarner ?